Google.Api.Generator.Tests/ProtoTests/Basic/Basic.proto

```diff
@@ -4,21 +4,14 @@ package testing.basic;
 
 // csharp_namespace option not included; to test generating namespace from proto package.
 
-import "google/api/annotations.proto";
 import "google/api/client.proto";
 
 service Basic {
   option (google.api.default_host) = "basic.example.com";
   option (google.api.oauth_scopes) = "scope1,scope2";
 
-  // Test summary text for IdempotentMethod
-  rpc IdempotentMethod(Request) returns(Response) {
```

Comment:
Would it be possible to split this PR into two commits, one of which changes to just use "AMethod" and then one for the gRPC service config part? At the moment I'm struggling to see which changes come from what.